docroot/themes/humsci/humsci_basic/src/scss/components/_masthead.scss

```diff
@@ -25,6 +25,49 @@
         margin-bottom: 0;
       }
     }
+
+    &__line1,
+    &__line2 {
+      font-size: hb-calculate-rems(24px);
+      font-weight: hb-theme-font-weight(semibold);
+    }
+
+    &__line3 {
+      font-size: hb-calculate-rems(19px);
+      font-weight: 500;
+    }
+
+    &__line5 {
+      font-size: 1.78rem;
+      text-transform: uppercase;
+      font-weight: hb-theme-font-weight(semibold);
+
+      /*Horizontal pipe on tablet and mobile */
+      @media (max-width: 767px) {
+        &::before {
+          margin-top: hb-calculate-rems(5px);
+          margin-bottom: hb-calculate-rems(7.5px);
+          border-bottom: $hb-thin-border;
+          border-color:#2e2d29;
+          content: "";
+          display: block;
+          width: 120px;
```

Comment:
I wasn't sure with this being a width if we wanted it converted to rem, so I opted to stay this route just in case. Kind of awaiting feedback to be sure. lol

It has been updated to use the rem calculate function.